recipes/knoten/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "knoten" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/knoten/archive/{{ version }}.tar.gz
+  sha256: d671f1c8c32107cf1ef1d2dadf236ba98f9f19dcfb744e302cb20860748164b0
```

Comment:
Is this a git hash? Is this in place of tagging for releases?

Reply:
This is the sha to check the integrity of the GitHub archive.